aca_hi_bgd/update_bgd_events.py

```diff
@@ -408,11 +417,16 @@ def get_manvr_extra_data(start: CxoTimeLike, stop: CxoTimeLike) -> dict:
         Dictionary of extra data for the dwell including "pitch" and "notes"
     """
 
-    pitchs = fetch.Msid("DP_PITCH", start, stop)
-    if len(pitchs.vals) > 0:
-        pitch = np.median(pitchs.vals)
-    else:
-        pitch = -999
+    # Explicitly set the fetch data source to the composite cxc/maude set, so that
+    # pitch_comp will just work (needs orbitephem not available to just maude source)
+    # For the high background monitor we could go back and use AOSARES1 if we
+    # wanted to exclude the cxc eng archive, but that seems not needed
+    with fetch.data_source("cxc", "maude allow_subset=False"):
```

Comment:
This is still allowing unexpected use of MAUDE. Sure, we don't plan right now to use it that way, and maybe the stop time is *supposed* to not let MAUDE get queried anyway if `--maude` is not supplied, but this still just seems against the intent of the PR to have a well-defined data source.

I had forgotten about the old-school `AOSARES1` solution. This is perfect for this application!